io-engine/src/core/snapshot.rs

```diff
@@ -33,8 +33,66 @@ impl SnapshotParams {
         }
     }
 }
-
-/// Snapshot Descriptor to respond back as part of listsnapshot
+/// Parameters details for the Snapshot Clone.
+#[derive(Clone, Debug)]
+pub struct CloneParams {
+    // Clone replica name
```

Comment:
Should use `///` and end with a `.`

Reply:
Accepted and Fix all the places in this file